gapis/gfxapi/templates/api.go.tmpl

```diff
@@ -1081,6 +1081,9 @@
     {{range $g := $.Globals}}
       {{Title $g.Name}} {{Template "Go.Type" $g}}
     {{end}}
+    SubcommandIndex gfxapi.SubcommandIndex `nobox:"true"`
```

Comment:
That sounds good, I don't remember reading such a comment, (although I may have just missed it). That is a lot nicer than what I have been doing, so I will do that.